### ISSUE TITLE
how to extract parameters of sim.batch_norm

### ISSUE THREAD

**Issue by patienceFromZhou** [tensorflow/tensorflow#14825]
using slim.batch_norm for normalize and here are the batch_norm_params:

![image](https://user-images.githubusercontent.com/31264567/33162735-1b13082c-d066-11e7-918c-62bec95e328c.png)

in this way, i think all the trainable variables (beta, gamma, moving_mean, moving_variance) was stored. and when i print elements in tf.trainable_variables, here is the result. 

![image](https://user-images.githubusercontent.com/31264567/33162835-8c4c091c-d066-11e7-96c9-da7f9b85865e.png)

missing gamma, 
i extracted the output tensor of the first layer, and manually calculate correspond feature map through these parameters.  its not the same, but can be transformed into the same through linear transformation.
so, i'm sure there's something wrong with batch_norm params. where can i find the correct ones.

**Comment by shivaniag** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
